.scripts/util.sh

```diff
@@ -63,52 +64,49 @@ is_generated() {
     fi
 }
 
-select_browser() {
+select_browser_caller() {
     local browser_engine="$1"
+    MSWEBVIEW_INCLUDE_DIR=$PACKAGE_DIR/Microsoft.Web.WebView2.$MSWEBVIEW_VERSION/build/native/include
+    MSWEBVIEW_LIB_DIR=$PACKAGE_DIR/Microsoft.Web.WebView2.$MSWEBVIEW_VERSION/build/native/x64
+    CHROMIUM_INCLUDE_DIR=$PACKAGE_DIR/chromiumembeddedframework.runtime.$CHROMIUM_VERSION/build/native/include
+    CHROMIUM_LIB_DIR=$PACKAGE_DIR/chromiumembeddedframework.runtime.$CHROMIUM_VERSION/build/native/x64
```

Comment:
No need to declare these here, rather deal with the values directly in the conditionals below